block_ai_chat.php

```diff
@@ -63,7 +63,7 @@ public function get_content(): stdClass {
         $this->content->footer = '';
 
         $context = \context_block::instance($this->instance->id);
-        if (!has_capability('block/ai_chat:view', $context)) {
+        if (!has_capability('block/ai_chat:view', $context) || !has_capability('local/ai_manager:use', $context)) {
```

Comment:
Müsste der Block ai_chat dann in dependencies nicht local_ai_manager stehen haben?

Reply:
yes it should, we have not done this yet in general for the plugins, but should  do at some point. Not scope of this ticket however, because of course the whole plugin heavily depends on local_ai_manager anyway.